golem-worker-service-base/src/service/worker/default.rs

```diff
@@ -127,6 +127,16 @@ pub trait WorkerService<AuthCtx> {
         auth_ctx: &AuthCtx,
     ) -> WorkerResult<WorkerMetadata>;
 
+    async fn get_metadatas(
+        &self,
+        template_id: &TemplateId,
+        filter: Option<WorkerFilter>,
+        cursor: u64,
+        count: u64,
+        precise: bool,
+        auth_ctx: &AuthCtx,
+    ) -> WorkerResult<(Option<u64>, Vec<WorkerMetadata>)>;
```

Comment:
What is u64 representing here as metadata?. 
I would  newtype it

Reply:
Also not sure what would it mean to `None` -> `WorkerMetadata`

Reply:
`Cursor`